src/pages/RecruitList/RecruitList.js

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect } from 'react';
+import { useInView } from 'react-intersection-observer';
+import * as S from './RecruitListStyled';
+import { BsFillBookmarkFill, BsChevronRight } from 'react-icons/bs';
+import { API } from '../../api';
+export default function RecruitList() {
+  const [list, setList] = useState([]);
+  const [scroll, setScroll] = useState(1);
+  const [ref, inView] = useInView();
+
+  useEffect(() => {
+    if (inView) {
+      setScroll(prev => prev + 1);
+    }
+  }, [inView]);
+  useEffect(() => {
```

Comment:
```suggestion
  }, [inView]);
  
  useEffect(() => {
```

여기도 한줄띄어서 구분 해 주시면 가독성에 좋겠습니다

Reply:
넵 수정했습니다